libs/@local/graph/migrations/graph-migrations/v009__entities/down.sql

```diff
@@ -7,3 +7,11 @@ DROP TABLE entity_temporal_metadata;
 DROP TABLE entity_editions;
 DROP TABLE entity_drafts;
 DROP TABLE entity_ids;
+
+DROP VIEW IF EXISTS first_entity_titles;
+DROP VIEW IF EXISTS last_entity_titles;
+DROP VIEW IF EXISTS entity_title_agg;
```

Comment:
Might be better to call this `first_entity_type_title` (and insert `type` everywhere else appropriate) as it correponds to the `type_title`. Entity's don't have titles, their types do.

e.g. also
> Corresponds to the first title of the [`Entity`].

->

> Corresponds to the title of the [`Entity`]'s first `EntityType`.

Reply:
I had that first but found that confusing as well (as it's the title of the entity's types). We could do `first_entity_type_title_for_entity_edition` but this is quite verbose. What do you think?

Reply:
`first_type_title_for_entity`?

Reply:
It think that's good!